amp/parse.go

```diff
@@ -56,6 +57,7 @@ func ParseParams(httpRequest *http.Request) (Params, error) {
 		Slot:            query.Get("slot"),
 		StoredRequestID: tagID,
 		Timeout:         parseIntPtr(query.Get("timeout")),
+		Targeting:       query.Get("targeting"),
```

Comment:
Can you add a non-empty targeting string to the `amp/parse_test.go#TestParseParams` test case `description: "All Fields"`?

Reply:
Good point, added!